### ISSUE TITLE
Allow to change batchExecutor's batchSize via command line

### ISSUE THREAD

**Issue by m417z** [jloosli/node-firestore-import-export#321]
Currently the import/export tasks are executed in parallel in batches of 50:
https://github.com/jloosli/node-firestore-import-export/blob/6977df5972cf84c8ef9ca6c527f57b2aad1f722b/src/lib/firestore-helpers.ts#L47

This number can't be changed via command line, only by modifying the source code.

50 is too large for my computer - the export task times out and fails. Changing it to 5 fixes the issue for me. It would be nice to have an option to control this number without having to fork the package.

**Comment by m417z** [comment]
> Changing it to 5 fixes the issue for me.

That wasn't true. I was probably just lucky that time. In fact, `batchExecutor` doesn't do its job at all, see https://github.com/jloosli/node-firestore-import-export/pull/346.

The issue is still relevant, though. Once it works, it can be useful to control the batch size limit via command line.
